airflow/contrib/hooks/aws_glue_job_hook.py

```diff
@@ -0,0 +1,214 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this the same file that is also added to `airflow/contrib/aws_glue_job_hook.py`?  If so, it should exist in only one location.

Reply:
@darrenleeweber, files in hooks,operator and specific folder are the correct ones(from my side atleast), should I delete others?